IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FlowInstrumentTypeElectricalCombination" UniqueId="8f68a974-45e4-30c7-5db5-c91cafca08a3" ApplicableType="IfcFlowInstrument/COMBINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AccuracyClass_32YEer4tP9EB633g5FK49p" />
+		<DocProperty xsi:nil="true" href="AccuracyGrade_3y6bNc8F56Hexr9vdk2L" />
+		<DocProperty xsi:nil="true" href="ApparentPower_1n4pozW90Wu_Sinh6vpNI" />
+		<DocProperty xsi:nil="true" href="NominalCurrent_1Sia_y4DvB595bUBLM1Dsb" />
+		<DocProperty xsi:nil="true" href="NominalPower_2pu16hiH1CqQv04BNgxAph" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_0KkDpffYz8L86gzG31TCSi" />
+		<DocProperty xsi:nil="true" href="PhaseNumber_1Lt0BWQy1AD811dqTjGf_d" />
+		<DocProperty xsi:nil="true" href="PrimaryCurrent_3exZs5G_1BZR8ePjN8RlDv" />
+		<DocProperty xsi:nil="true" href="PrimaryFrequency_3bDvHjEfEAwV9_ElCeTdI" />
+		<DocProperty xsi:nil="true" href="PrimaryVoltage_2vDDOIIxP0uSbVrMQcOKz" />
+		<DocProperty xsi:nil="true" href="SecondaryCurrent_0BPcq9v3b1rOX4CuGa6OPK" />
```

Comment:
using different versions of secondary current this is the created one

Reply:
Done. Commit : 0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f

Reply:
You missed it... :smile: